code/modules/clothing/under/accessories/storage.dm

```diff
@@ -5,6 +5,7 @@
 	_color = "webbing"
 	var/slots = 3
 	var/list/can_only_hold = list() //I would add the other storage item variables, but nothing would use them yet, so there's no point.
+	var/list/cant_hold = list("/obj/item/clothing/accessory/storage", "/obj/item/clothing/under") //NO RECURSION
```

Comment:
That's built into `cant_hold`
It's only a strict type check if there's an = at the beginning of the string
Otherwise, it checks `typesof()`